src/index.d.ts

```diff
@@ -22,17 +22,16 @@ import {
   TextInputProps,
   ImageProps as RNImageProps,
   TouchableHighlightProps,
-  PressableProps,
 } from 'react-native';
 import { RatingProps, AirbnbRatingProps } from 'react-native-ratings';
 import {
   IconButtonProps,
   IconProps as VectorIconProps,
 } from 'react-native-vector-icons/Icon';
 
-export interface TouchableComponent
-  extends TouchableHighlightProps,
-    PressableProps {}
+export interface TouchableComponent extends TouchableHighlightProps {
+  children?: React.ReactElement<any>;
```

Comment:
this gives a type error, unless there's only a single child (such as wrapping the children into `React.Fragment`). Just removing the `children` entry works.

(Happy you removed `PressableProps` though as we had to patch it (we're on RN 0.62 🙂 ))

Reply:
`children` is correctly inherited via https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6a47fa33adb7c55be63bbca45ce86d0ff4e59b18/types/react/index.d.ts#L501, so I don't think you need to specify this at all?